Applications/MainApplication/Gui/MainWindow.cpp

```diff
@@ -540,6 +549,19 @@ void MainWindow::resetScene() {
     fitCamera();
 }
 
+void MainWindow::useCamera() {
+    if ( m_selectionManager->hasSelection() )
+    {
+        const ItemEntry& ent = m_selectionManager->currentItem();
+        Engine::Camera* camera = dynamic_cast<Engine::Camera*>( ent.m_component );

```

Comment:
Prefer a static_cast after being sure that the component is a camera (see comment above)